src/settings/components/Profile.tsx

```diff
@@ -100,9 +101,9 @@ const Profile: FunctionComponent<
 	const [selectedOrganisations, setSelectedOrganisations] = useState<
 		ClientEducationOrganization[]
 	>(get(user, 'profile.organizations', []));
-	const [firstName, setFirstName] = useState<string>(get(user, 'first_name') || '');
-	const [lastName, setLastName] = useState<string>(get(user, 'last_name') || '');
-	const [alias, setAlias] = useState<string>(user ? getProfileAlias(user) : '');
+	const [firstName] = useState<string>(get(user, 'first_name') || '');
+	const [lastName] = useState<string>(get(user, 'last_name') || '');
+	const [email] = useState<string>(get(user, 'mail') || '');
```

Comment:
I think we can remove the `useState` and simply store the values in a const. The empty fallback string kan also be set in the lodash get function.

```suggestion
	const firstName = get(user, 'first_name', '');
	const lastName = get(user, 'last_name, '');
	const email = get(user, 'mail, '');
```

Reply:
agreed, but lets use elvis operators instead of lodash get function since it provides better type safety
```
	const firstName = user?.first_name || '';
	const lastName = user?.last_name || '';
	const email = user?.mail || '';
```